### PR TITLE
Added React.js Course By Mosh

### PR DESCRIPTION
**Pull request by prerna120**
## What does this PR do?
Added react.js course

## For resources
Youtube channel: https://www.youtube.com/c/programmingwithmosh
Youtube video: https://www.youtube.com/watch?v=Ke90Tje7VS0
### Description
React.js course for beginners

### Why is this valuable (or not)?
It is one of the best courses to learn react.js main concepts in minimum time.

### How do we know it's really free?
It's on youtube

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -654,6 +654,7 @@
 * [ReactJS Frontend Web Development For Beginners](https://www.udemy.com/course/react-tutorial/) - Ryan Dhungel (Udemy)
 * [ReactJS Full Course for Beginners \| Complete All-in-One Tutorial \| 9 Hours](https://www.youtube.com/watch?v=RVFAyFWO4go) - Dave Gray
 * [ReactJS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3) - Codevolution
+* [ReactJS-Tutorial for Beginners](https://www.youtube.com/watch?v=Ke90Tje7VS0) - Programming with Mosh
```
Comment (eshellman) [inline courses/free-courses-en.md:657]: For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

Comment (prerna120) [reply courses/free-courses-en.md]: Hi eric, i have done the suggested changes.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -654,6 +654,7 @@
 * [ReactJS Frontend Web Development For Beginners](https://www.udemy.com/course/react-tutorial/) - Ryan Dhungel (Udemy)
 * [ReactJS Full Course for Beginners \| Complete All-in-One Tutorial \| 9 Hours](https://www.youtube.com/watch?v=RVFAyFWO4go) - Dave Gray
 * [ReactJS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3) - Codevolution
+* [ReactJS-Tutorial for Beginners](https://youtu.be/Ke90Tje7VS0) - Mosh Hamedani
```
Comment (eshellman) [inline courses/free-courses-en.md:657]: For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes
NO SHORTENED LINKS!

Comment (prerna120) [reply courses/free-courses-en.md:657]: It is not shortened link. The course is not in playlist it's one video course. That's why it's link is in this form.😅

Comment (davorpa) [reply courses/free-courses-en.md:657]: Expand YT urls

```suggestion
* [ReactJS - Tutorial for Beginners](https://www.youtube.com/watch?v=Ke90Tje7VS0) - Mosh Hamedani
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -689,6 +689,7 @@
 * [React Crash Course 2020](https://www.youtube.com/watch?v=4UZrsTqkcW4) - freecodecamp
 * [React Hooks](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH8EtggFGERCwMY5u5hOjf-h) - WebDevSimplified
 * [React Tutorials](https://www.youtube.com/playlist?list=PLWKjhJtqVAbkArDMazoARtNz1aMwNWmvC) - freeCodeCamp
+* [ReactJS - Tutorial for Beginners](https://www.youtube.com/watch?v=Ke90Tje7VS0) - Mosh Hamedani
```
Comment (davorpa) [inline courses/free-courses-en.md:692]: Complete instructor platform

```suggestion
* [ReactJS - Tutorial for Beginners](https://www.youtube.com/watch?v=Ke90Tje7VS0) - Mosh Hamedani (Programming with Mosh)
```

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-en.md (hunk 2 above)
Comment (prerna120) [reply courses/free-courses-en.md]: Hi David, thank you for the explaination. Initially, i did the same as the change suggested by you. After the Eric's comment,i changed it. I was confused, sorry for the inconvenience caused.

**Pull request merged**